pandas/tests/io/test_excel.py

```diff
@@ -1000,6 +1000,20 @@ def test_read_excel_skiprows_list(self):
                                'skiprows_list', skiprows=np.array([0, 2]))
         tm.assert_frame_equal(actual, expected)
 
+    def test_read_excel_nrows(self):
+        # GH 16645
+        num_rows_to_pull = 5
+        actual = pd.read_excel(os.path.join(self.dirpath, 'test1' + self.ext),
+                               nrows=num_rows_to_pull)
+        expected = pd.read_excel(os.path.join(self.dirpath,
+                                              'test1' + self.ext))
+        expected = expected[:num_rows_to_pull]
+        tm.assert_frame_equal(actual, expected)
+
+        with pytest.raises(ValueError):
```

Comment:
1) We have an entire test suite for this in `common.py` (under `io/parsers`)

2) This already gets validated.  Thus, this test is somewhat unnecessary.